sdk/core/azure-core-amqp/CHANGELOG.md

```diff
@@ -4,8 +4,12 @@
 
 ### Features Added
 
+The `Close` method on AMQP Message Sender and Message Receiver now blocks until the client receives a `DETACH` response from the remote node.
+
 ### Breaking Changes
 
+- The `Close` method on AMQP Message Sender and Message Receiver now accepts an `Azure::Core::Context` parameter. This parameter is defaulted so this change should not affect existing code.
```

Comment:
This change is in `_internal` surface, so I think you don't have to add this changelog line.

Reply:
Azure Core has changes to the _internal surface in the changelog, why is AMQP different?

Reply:
Does it? The same comment applies there then.

Reply:
Finally, up to you as an author. All I said, "you don't have to".

Reply:
For AMQP, the entire functional API surface is in _internal, so I think it belongs.

And yes, for instance we have documented changes to the logging infrastructure, which is all internal (changing the format of log strings, updating log levels for individual messages, etc).